src/playground/with-layout/stepper/stepper-step-change-event.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component } from '@angular/core';
+
+@Component({
+  templateUrl: './stepper-step-change-event.component.html',
+  styles     : [`.selected-index {
+      font-size: 24px;
```

Comment:
Please, use font helper classes provided by Nebular. `.h4` could fit here.
We can't use own random font settings as we should follow design system typography rules.